apps-prd/us-east-1/security-certs/config.tf

```diff
@@ -2,9 +2,11 @@
 # AWS Provider Settings       #
 #=============================#
 provider "aws" {
-  region                  = var.region
-  profile                 = var.profile
-  shared_credentials_file = "~/.aws/${var.project}/config"
+  region  = var.region
+  profile = var.profile
+  # comment this 2 lines for aws sso enable
+  # shared_credentials_files = ["~/.aws/${var.project}/credentials"]
+  # shared_config_files      = ["~/.aws/${var.project}/config"]
```

Comment:
@rodriguez-matias Matu let's remove them, this lines are not necessary even for SSO disabled. Thanks!

Reply:
OK got it. I'll remove them all on each layer.